### ISSUE TITLE
Gamepad multitouch API

### ISSUE THREAD

**Issue by bradleyneedham** [WebKit/standards-positions#110]
### Information about the spec

* Spec Title: GamepadTouch Interface
* Spec URL: https://knyg.github.io/gamepad/extensions.html#gamepadtouch-interface
* GitHub repository: https://github.com/w3c/gamepad/
* Issue Tracker (if not the repository's issue tracker): https://chromestatus.com/feature/4782975812108288
* Explainer (if not README.md in the repository): https://github.com/bradleyneedham/gamepad/blob/multitouch-explainer/multitouch_explainer.md

### Design reviews and vendor positions

* TAG Design Review: https://github.com/w3ctag/design-reviews/issues/799#issuecomment-1353669879
* Mozilla standards-positions issue: https://github.com/mozilla/standards-positions/issues/724

### Bugs tracking this feature

* Chromium Bugs: https://bugs.chromium.org/p/chromium/issues/detail?id=1143942

Brief summary of the issue or the subject of the spec.
Addition of new web interfaces that provide web developers with access to the touchpad inputs from modern Gamepads.

**Comment by marcoscaceres** [comment]
@bradleyneedham, I don't think we are opposed to this, but we should really move the out of the extension spec into the spec proper (and use that to gauge implementer interest and commitment). I see there has been some movement already on the Chrome bug - what's the status there? 

From a spec perspective, it's missing all the details about how/when the events are fired. The proposal also needs some wording around `id`s to make they remain unique per origin to prevent fingerprinting. 

There probably needs to be some explanation about the overlap with touch events too. I see Rick Byers argued in the TAG issue that they are "different", but it would be great to go into the details as to why they are different.

**Comment by bradleyneedham** [comment]
@marcoscaceres thank you for the comment. The windows implementation on chrome is almost done, I am currently trying to get it committed.

Touch events like other Gamepad data are pulled rather than fired to a callback at the application layer. I will see if I can make this clearer in the spec as well as clarify the uniqueness per origin of the `id`s.
